config/peerpods/podvm/lib.sh

```diff
@@ -240,7 +240,7 @@ function prepare_source_code() {
 
 }
 
-# Download and extract pause container image
+# Drive download and extract of the pause container image
```

Comment:
Any specific reason to use "Drive download" ?

Reply:
reverted, idea was that as there is another function which does the actual download & extract, show this as a function which drives the download of pause image. but yeah I think old one sounds good..